web/main.js

```diff
@@ -5,6 +5,18 @@ import { genpassword } from './wordlist.js';
 let receiving;
 let sending;
 let datachannel;
+let downloadServiceWorker; // Service worker managing `/download/...` urls.
+
+const serviceWorkerInUse = 'serviceWorker' in navigator;
+
+if (serviceWorkerInUse) {
+	navigator.serviceWorker.register('sw.js', {
+		scope: '/download/'
```

Comment:
let's change this to something weirder (i'm leaning towards "/_/") so it wouldn't be confused with a reasonable url.

we can also add a handler for this in the go server, so if someone ends up there somehow they would get a nicer message asking them to file an issue. no need to do that in this pr though.

(in the future i'd like to provide command line binaries at /download/)